processor/processor.go

```diff
@@ -1007,6 +1008,14 @@ func getDiffMetrics(inPU, pu string, inCountMetadataMap map[string]MetricMetadat
 	//diff = succesCount + abortCount - inCount
 	diffMetrics := make([]*types.PUReportedMetric, 0)
 	for key, inCount := range inCountMap {
+		var eventName, eventType string
+		splitKey := strings.Split(key, METRICKEYDELIMITER)
+		if len(splitKey) < 5 {
+			eventName = ""
+			eventType = ""
+		}
+		eventName = splitKey[3]
```

Comment:
We will try to access this even when the length of splitkeys is less than 5? Shouldn't this be in else case